src/plugins/UrbanDictionary.js

```diff
@@ -0,0 +1,46 @@
+const Plugin = require('../Plugin');
+const request = require('request-promise');
+
+module.exports = class UrbanDictionary extends Plugin {
+    static get plugin() {
+        return {
+            name: "UrbanDictionary",
+            description: "Fetches the Urban Dictionary definition for a supplied word or phrase.",
+            help: ` commands:
```

Comment:
For consistency, I think it's better if you only include the list of commands, without the "commands:" header.